vnext/Desktop.IntegrationTests/DesktopTestRunner.cpp

```diff
@@ -73,6 +94,10 @@ shared_ptr<ITestInstance> TestRunner::GetInstance(
       make_tuple(
           TestDeviceInfoModule::name,
           []() -> unique_ptr<CxxModule> { return make_unique<TestDeviceInfoModule>(); },
+          nativeQueue),
+      make_tuple(
+          "StatusBarManager",
+          []() -> unique_ptr<CxxModule> { return make_unique<StubNativeModule>("StatusBarManager"); },
```

Comment:
>StatusBarManager [](start = 80, length = 16)

ReactUwp/Microsoft.ReactNative have a stub version of this we should consider moving to ReactWindowCore, but this is all going to change and I have reworking Windesktop based tests on my plate to do after we're continually on master anyway.